package.json

```diff
@@ -2,6 +2,9 @@
   "name": "lol",
   "type": "module",
   "version": "0.0.1",
+  "bin": {
+    "loam": "./node_modules/.bin/loam"
+  },
```

Comment:
```suggestion
```
The bin should just bin in the `loam-cli`'s `package.json`. Sorry if I was unclear before.

Reply:
No problem, I got there eventually! :) 

I still think that we need to run `npx loam` rather than just `loam` since the `node_modules/.bin` dir may not be on the user's path. But that is probably still better than a global install!